File: athrill-account-client/projects/at-models/src/lib/users/user-login.model.ts
import { BaseModel } from '../at-base.model';
import { UserModel } from './user.model';

export class UserLoginModel extends BaseModel
{
    public UserLoginId : number;
    public UserId : number;
    public Username : string;
    public IsResetPassword? : boolean;

    public User : UserModel;

    public Id : number;
}

File: athrill-account-server/Libraries/Model/Users/UserLoginModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AT.Model.Users
{
    public class UserLoginModel : BaseModel
    {
        public int UserLoginId { get; set; }
        public int UserId { get; set; }
        public string Username { get; set; }
        public bool? IsResetPassword { get; set; }

        #region Link 
        public  UserModel User { get; set; }
        #endregion

        #region override
        public override int Id => UserLoginId;
        public override string ToString()
        {
            return Username;
        }
        #endregion
    }
}
